tests/__init__.py

```diff
@@ -9,8 +9,11 @@
     SAMPLE_PUBLIC_KEY,
 )
 
+# JWT related test config
 os.environ["AUTH_REMOVE_PREFIXES"] = "1"
 os.environ["ALLOWED_PUBLIC_KEYS"] = json.dumps({"keys": [SAMPLE_PUBLIC_KEY]})
 os.environ["ALLOWED_AUDIENCES"] = ",".join([str(uuid4()), str(uuid4())])
 os.environ["ALLOWED_ISS"] = "test-issuer"
+# AWS related test config
 os.environ["AWS_LAMBDA_FUNCTION_NAME"] = "million-dollar-lambda"
+os.environ["AWS_DEFAULT_REGION"] = "us-west-2"
```

Comment:
[#961430505](https://github.com/pdyba/lambdalizator/pull/63#discussion_r961430505):
> It would be better to set these all env variables with the usage of the fixture.
> 
> But you can place TODO to do it in the future if you wish 😉